test/test_transforms_v2_refactored.py

```diff
@@ -1634,3 +1636,56 @@ def test_transform_negative_degrees_error(self):
     def test_transform_unknown_fill_error(self):
         with pytest.raises(TypeError, match="Got inappropriate fill arg"):
             transforms.RandomAffine(degrees=0, fill="fill")
+
+
+class TestCompose:
+    class BuiltinTransform(transforms.Transform):
+        def _transform(self, inpt, params):
+            return inpt
+
+    class PackedInputTransform(nn.Module):
+        def forward(self, sample):
+            image, label = sample
+            return image, label
+
+    class UnpackedInputTransform(nn.Module):
+        def forward(self, image, label):
+            return image, label
+
+    @pytest.mark.parametrize(
+        "transform_clss",
```

Comment:
Why not call this `transform_class` ?

Reply:
It would have to be `transform_classes`, since it is a list of classes. And since we use `cls` for singular, I'm usually just append an `s` to it. I'll leave it up to you.